classes/decision/types/Decline.inc.php

```diff
@@ -32,6 +33,7 @@
 class Decline extends DecisionType
 {
     use InExternalReviewRound;
+    use withReviewRound;
```

Comment:
this is because some of the code from the trait `InExternalReviewRound` moved into the trait `withReviewRound` and that is why it was added to those decisions .